addons/sourcemod/scripting/tfgo.sp

```diff
@@ -539,7 +539,7 @@ void PlantBomb(TFTeam team, int cpIndex, ArrayList cappers)
 	for (int i = 0; i < cappers.Length; i++)
 	{
 		int capper = cappers.Get(i);
-		TFGOPlayer(capper).AddToAccount(tfgo_cash_player_bomb_planted.IntValue, "%T", "Player_Cash_Award_Bomb_Planted", LANG_SERVER, tfgo_cash_player_bomb_planted.IntValue);
+		TFGOPlayer(capper).AddToAccount(tfgo_cash_player_bomb_planted.IntValue, "%t", "Player_Cash_Award_Bomb_Planted", tfgo_cash_player_bomb_planted.IntValue);
```

Comment:
`TFGOPlayer.AddToAccount` Should use `SetGlobalTransTarget` to set language as target `this`